assets/terraform-modules/controller/main.tf

```diff
@@ -46,6 +46,7 @@ data "ct_config" "config" {
     kubelet_image_name        = var.kubelet_image_name
```

Comment:
Little NIT, I think conroller is set to `<cluster-name>-controller-<index>` but commit message specify `<cluster-name>.controller-<index>`

Reply:
Whats the difference here @knrt10 ?

Reply:
code has a dash and commit has a dot

Reply:
ah :facepalm: